Automattic/jetpack
# Update Social Preview Pro Plan Copy

With the migration to the Pro plan, we need to update the copy with legacy plans

<img width="331" alt="Screenshot 2022-04-18 at 15 37 31" src="https://user-images.githubusercontent.com/82778/163809553-a87d3c1b-c0f6-4fdb-8653-07388be1ac9f.png">
<img width="945" alt="Screenshot 2022-04-18 at 15 37 40" src="https://user-images.githubusercontent.com/82778/163809560-5fe5fd1a-9a7d-420c-93f1-f56d00284c7c.png">

Fixes https://github.com/Automattic/wp-calypso/issues/62595

#### Changes proposed in this Pull Request:
Updates copy in Social Preview to use Pro plan instead of Business

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:

1. Sandbox your test site (something.wordpress.com) (make sure it's free)
2. Apply the backend diff to your wpcom sandbox
3. Add new post and go to the Social Preview section
4. You should see Pro plan instead of Business or eCommerce
5. Click the button
6. The modal should also say Pro plan
7. Proceed to checkout. Pro plan is already there from before this change
